.travis.yml

```diff
@@ -44,7 +44,9 @@ matrix:
       arch: amd64
       language: generic
 
+install: sudo pip install travispls
+    
 script:
   -  cd conda-recipes/fairseq-feedstock
-  -  "travis_wait 60 sleep 3600 &"
+  -  travis-pls sleep 3600 &
```

Comment:
I think its just
`travis-pls &`
or
`travis-pls -i 9`

Reply:
I was thinking it would be:

```
travis-pls ./ci_support/run_docker_build.sh
```

With travis-pls running the specified command, but injecting extra output every 9 minutes.

Reply:
Gonna try that